docs/pages/Architecture/Discovery and Peering/discovery-component.md

```diff
@@ -3,7 +3,31 @@ title: "Discovery Protocol"
 ---
 
 ## Overview
-This component's goal is to find other clusters running Liqo around us, get information needed to pair and start peering process
+The goal of this component is to find other clusters running Liqo on the same LAN, and to obtain the information required to possibly start the peering process.
+
+
+## Peering Process
```

Comment:
Why are we presenting the peering process *before* the neighbor discovery? Shouldn't be the opposite?